src/g_shared/a_quake.cpp

```diff
@@ -95,6 +98,22 @@ void DEarthquake::Serialize (FArchive &arc)
 	{
 		arc << m_WaveSpeedX << m_WaveSpeedY << m_WaveSpeedZ;
 	}
+	if (SaveVersion < 4525)
+	{
+		m_Falloff <<= FRACBITS;

```

Comment:
Yes.

https://github.com/rheit/zdoom/blob/master/src/g_shared/a_quake.cpp#L72

Randi had it set up that way too.